nailgun/entities.py

```diff
@@ -847,6 +847,57 @@ def update(self, fields=None):
         return self.read()
 
 
+class VMWareComputeResource(AbstractComputeResource):
```

Comment:
Please move these 2 classes right after LibvirtComputeResource in opposite order to achieve alphabetical order like
```
class DockerComputeResource(AbstractComputeResource):
class LibvirtComputeResource(AbstractComputeResource):
class OVirtComputeResource(AbstractComputeResource):
class VMWareComputeResource(AbstractComputeResource):
```